hpx/runtime/serialization/basic_archive.hpp

```diff
@@ -57,7 +57,7 @@ namespace hpx { namespace serialization
         {}
 
         template <typename T>
-        void invoke(T * t)
+        void invoke(T * (&t))
         {
```

Comment:
This change will cause this specialization of invoke() to only match pointers (e.g. int *foo), not arrays (e.g. int foo[30]). I need to serialize member arrays in LGD as many simulation models contain plain arrays (instead of std::vector or smart pointers).